charm/src/iframe/prompt.ts

```diff
@@ -31,21 +31,20 @@ export const buildPrompt = ({
   schema: JSONSchema;
   model?: string;
 }): LLMRequest => {
-  const messages = [];
+  const messages: string[] = [];
   if (spec && src) {
     messages.push(spec);
```

Comment:
I think this should say something like "the user asked you to create `spec`" -- perhaps